psse/psse-model/src/main/java/com/powsybl/psse/model/pf/PsseValidation.java

```diff
@@ -204,6 +209,18 @@ private void validateNonTransformerBranches(List<PsseNonTransformerBranch> nonTr
         checkDuplicatesLinks("NonTransformerBranch", "branches", getDuplicates(busesNonTransformerBranches));
     }
 
+    private boolean isBaldyConnectedNonTransformedBranch(PsseNonTransformerBranch nonTransformerBranch, Map<Integer, List<Integer>> buses) {
+        if (!buses.containsKey(nonTransformerBranch.getI())) {
```

Comment:
This is not conditionned by the fact the transformer is out of service. Is it on purpose ? Is it what PSS/E is doing ?

Reply:
Yes it is on purpose. If the bus is not well defined, PSS/E discards the record in both cases, whether the status is 0 or 1..

Reply:
Ok, good to known